src/components/view/DocumentListPage.js

```diff
@@ -0,0 +1,81 @@
+import DocumentList from "../common/DocumentList.js";
+import { request } from "../../js/api.js";
+import { push } from "../../router/router.js";
+
+export default function DocumentListPage({ $bodyPage, initialState }) {
+  const $documentListPage = document.createElement("div");
+  $documentListPage.classList.add("list-container");
+
+  // by 민형, Editor에서 입력할 때 마다 initialState에 수정된 데이터가 전달됨_221116
+  // Editor에 입력시 list에 바로 반영되는 기능 구현에만 사용
+  this.documentListState = initialState;
+
+  this.documentListSetState = (nextState) => {
+    this.documentListState = nextState;
+    this.sendData();
+    this.render();
+  };
+
+  const documentList = new DocumentList({
+    $documentListPage,
+    initialState: { originEdit: [], updateEdit: {} },
```

Comment:
p3;
sendData() 를 통해서 documentList의 updateEdit state를 변경하는 것으로 보입니다. 상위 컴포넌트에서 state를 전달받아야 한다면 아래 처럼 작성하는 것이 데이터의 흐름을 파악하는데 더 효과적일 것 같습니다~

```suggestion
    initialState: { originEdit: [], updateEdit: this.documentListState.updateEdit },
```